indico/modules/rb/models/locations.py

```diff
@@ -49,17 +54,20 @@ def working_time_end(cls):
         db.Integer,
         primary_key=True
     )
+
```

Comment:
i think we usually don't have empty lines there

Reply:
Which is a bit weird, cause we don't have spaces between columns but then add them between relationships. Go figure.

Reply:
I think in newer models we are consistent:

- no whitespace between columns
- no whitespace between relationships
- whitespace between [...columns...]/[...relationships...]

Reply:
Can we document that somewhere? Some code style wiki page or something for stuff that is not done by flake8/ESLint. I think you're the right guy to do that 😉